Aprendizado por Reforço Profundo/Deep Q-Learning/Prioritized Experience Replay/Prioritized Experience Replay.ipynb

```diff
@@ -0,0 +1,353 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "LdvU1bXObNVc"
+   },
+   "source": [
+    "# Prioritized Experience Replay\n",
+    "\n",
+    "Baseado no paper [Prioritized Experience Replay, Schaul et al, 2015.](https://deepmind.com/research/publications/prioritized-experience-replay)"
```

Comment:
A ideia é a explicação ficar só no README?

Reply:
A gente tava chegando no padrão de colocar o mesmo texto no readme e nb, certo? Tava pensando em fazer tudo no readme e quando tiver pronto mover pro nb (ou adaptar, se for o caso)

Reply:
Beleuza